src/objects/button-group/button-group.scss

```diff
@@ -0,0 +1,38 @@
+@use "../../compiled/tokens/scss/breakpoint";
+@use "../../compiled/tokens/scss/size";
+
+.o-button-group {
+  display: flex;
+  flex-direction: column;
+  gap: size.$button-group-gap-default;
```

Comment:
@gerardo-rodriguez do you think it would make sense to use margin on the items + negative margin on the container instead of gap for flexbox? iOS 14.5 is the first iOS version to support gap for flexbox, and it is only 1 month old. [StatCounter](https://gs.statcounter.com/ios-version-market-share/mobile-tablet/worldwide) doesn't even have data on iOS 14.5 usage yet since their data lags by a month

Reply:
Also the margin + negative margin looks/works the same

Reply:
Ooh just realized you could use a `--gap` custom property + `calc` to make a sort of "polyfill" for this 🤯

Reply:
Oh, did you find an example of this somewhere? 🔍 👀

Reply:
No I just thought of it:


```css
.parent {
  display: flex;
  --gap: 10px;
  margin: calc(- var(--gap, 0) / 2)
}
.parent.tight {
  --gap: 5px;
}

.child {
  margin: calc(var(--gap, 0) / 2);
}
```

Reply:
@calebeby Oh, great idea! 🎉

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1240/commits/5428bd1382d1b33c938cf7e7dd00a3a1c07c11ae